src/containers/LandingPage/FallbackPage.js

```diff
@@ -7,69 +7,16 @@ export const fallbackSections = {
   sections: [
     {
       sectionType: 'features',
-      sectionId: 'hero',
-      background: { type: 'hexColor', color: '#ffff00' },
-      // backgroundImage: {
-      //   type: 'image',
-      //   alt: 'Background image',
-      //   image: {
-      //     id: 'image',
-      //     type: 'imageAsset',
-      //     attributes: {
-      //       variants: {
-      //         square1x: {
-      //           url: `https://picsum.photos/400/400`,
-      //           width: 400,
-      //           height: 400,
-      //         },
-      //         square2x: {
-      //           url: `https://picsum.photos/800/800`,
-      //           width: 800,
-      //           height: 800,
-      //         },
-      //       },
-      //     },
-      //   },
-      // },
-      blocks: [
-        {
-          blockType: 'default-block',
-          blockId: 'hero-content',
-          media: {
-            type: 'image',
-            alt: 'First image',
-            image: {
-              id: 'image',
-              type: 'imageAsset',
-              attributes: {
-                variants: {
-                  square1x: {
-                    url: `https://picsum.photos/400/400`,
-                    width: 400,
-                    height: 400,
-                  },
-                  square2x: {
-                    url: `https://picsum.photos/800/800`,
-                    width: 800,
-                    height: 800,
-                  },
-                },
-              },
-            },
-          },
-          title: { type: 'heading1', content: 'My marketplace' },
-          text: {
-            type: 'markdown',
-            content:
-              '### My unique marketplace for booking listings\n### You can also list your services here!',
-          },
-          callToAction: {
-            type: 'internalButtonLink',
-            href: '/s',
-            label: 'Browse marketplace',
-          },
-        },
-      ],
+      sectionId: 'maintenance-mode',
+      title: {
+        type: 'heading1',
+        content: 'HTTP 500 Internal Server Error',
```

Comment:
Ei tää ole Internal server error.

Kirjoita mieluummin jotain tyyliin:
"Maintenance mode" / "The marketplace is not fully operational at the moment." 
Ja ehkä vielä: "Try refreshing the page and if that doesn't solve the issue, contact the marketplace admins."

Reply:
You could fine-tune this more. E.g. use an image or something in that direction.
For the Sharetribe-hosted end-users, it will communicate that the marketplace is not fully functional at the moment.
(This should be a very rare event, but it's still possible that assets are not working properly.)